cmd/configschema/go.mod

```diff
@@ -53,10 +53,10 @@ require (
 	github.com/ClickHouse/clickhouse-go/v2 v2.3.0 // indirect
 	github.com/DataDog/agent-payload/v5 v5.0.34 // indirect
 	github.com/DataDog/datadog-agent/pkg/obfuscate v0.40.0-rc.2.0.20220930142800-72e24cd2e404 // indirect
-	github.com/DataDog/datadog-agent/pkg/otlp/model v0.40.0-rc.2.0.20220930142800-72e24cd2e404 // indirect
-	github.com/DataDog/datadog-agent/pkg/quantile v0.40.0-rc.2.0.20220930142800-72e24cd2e404 // indirect
+	github.com/DataDog/datadog-agent/pkg/otlp/model v0.40.0-rc.2.0.20221013201707-c46e9de16283 // indirect
```

Comment:
why not https://github.com/DataDog/datadog-agent/releases/tag/pkg%2Fotlp%2Fmodel%2Fv0.40.0-rc.5?

Reply:
It doesn't include the upgrade to 0.62.0

Reply:
v0.40.0-rc.5 was tagged on the 7.40.x branch. The 0.62.0 bump will not be available on 7.40, but on 7.41. The pseudoversion is a bit confusing...